src/Makefile

```diff
@@ -21,22 +21,35 @@ CL_LIB=cl-crypt
 
 CHACHA_IF_DIR:=crypt-if
 
-CL_HEADER_DIR:=opencl-platform
+# for Windows NVIDIA cygwin sample
```

Comment:
Remove commented lines or do some kind of platform detection.

Reply:
Just wanted to give some examples, but maybe these would be better placed in the Readme ?